core/templates/services/UpgradedServices.ts

```diff
@@ -526,6 +526,7 @@ import { SolutionVerificationService } from
   // eslint-disable-next-line max-len
   'pages/exploration-editor-page/editor-tab/services/solution-verification.service';
 import { QuestionValidationService } from './question-validation.service';
+import { ExplorationCreationService } from 'components/entity-creation-services/exploration-creation.service';
```

Comment:
No new additions are allowed in UpgradedServices.ts

Reply:
done